aws/src/handlers/create-feedback.js

```diff
@@ -0,0 +1,68 @@
+const { Client } = require("pg");
+
+const allowedOrigins = [
+  "https://docs.aiven.io",
+  "http://localhost:[0-9]*",
+  "https://devportal.pages.dev",
```

Comment:
Need to include preview branch url. eg: https://e7e0d3e1.devportal.pages.dev/
https://*.devportal.pages.dev